src/bin/rustdoc-ember.rs

```diff
@@ -0,0 +1,85 @@
+#[macro_use]
```

Comment:
what do you think about making this a workspace, with `rustdoc-ember` as a crate inside of it? This would let us get rid of the build script for the main `rustdoc` crate, which feels cleaner to me.

Reply:
I'd prefer a workspace as well, but I don't think it's possible to force all crates be built? I'm trying to avoid the situation where you build the backend and then the frontend can't be found. If that's acceptable, I'm happy to make it into a separate crate, though.

Reply:
It is, IMO.